lldb/include/lldb/API/SBProcess.h

```diff
@@ -407,6 +407,117 @@ class LLDB_API SBProcess {
   ///     the process isn't loaded from a core file.
   lldb::SBFileSpec GetCoreFile();
 
+  /// \{
+  /// \group Mask Address Methods
+  ///
+  /// \a type
+  /// All of the methods in this group take \a type argument
+  /// which is an AddressMaskType enum value.
```

Comment:
Can our Doxygen system make a reference to the documentation for these named constants? Just a question.

Reply:
I need to read the Doxygen documentation more closely, I just made this up when I was trying to update to having references to these two documentation paragraphs, I don't know if it would render as anything meaningful.